LondonBritishLibrary/orient/BLorient6466.xml

```diff
@@ -0,0 +1,427 @@
+                                        <title ref="LIT2265salota"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.7">
+                                        <locus from="154r" to="154v"/>
+                                        <title ref="LIT2274salota"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.8">
+                                        <locus from="154v" to="156r"/>
+                                        <title ref="LIT2250salota"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.9">
+                                        <locus target="#156r"/>
+                                        <title ref="LIT2271salota"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.10">
+                                        <locus from="156r" to="157r"/>
+                                        <title ref="LIT1566hababa"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.11">
+                                        <locus from="157r" to="159r"/>
+                                        <title ref="LIT2251salota"/>
+                                        <incipit xml:lang="gez"><title ref="LIT2251salota">ትንቢት፡ ዘዕንባቆም፡ ነገር፡ ዘክርስቶስ፡ ወዘ፡ ዘሩባቤል።</title></incipit>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.12">
+                                        <locus from="159r" to="160r"/>
+                                        <title ref="LIT2259salota"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.13">
+                                        <locus target="#160r"/>
+                                        <title ref="LIT1827Magnif"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.14">
+                                        <locus from="160v" to="161r"/>
+                                        <title ref="LIT2275salota"/>
+                                    </msItem>
+                                    <msItem xml:id="p3_i1.2.1.15">
+                                        <locus target="#161r"/>
+                                        <title ref="LIT2272salota"/>
+                                    </msItem>
+                                </msItem>
+                                <msItem xml:id="p3_i1.2.2">
+                                    <locus from="161r" to="169v"/>
+                                    <title ref="LIT2362Songof"/>
+                                    <note>The text is divided into five sections, all but the first of which are preceded by the following doxology: 
```

Comment:
Do these sections usually correspond to those of the edition encoded as textparts in LIT2362Songof (FirstCanticle etc)?

Reply:
I have not checked this. In any case these five sections regularly appear in alle descriptions on Psalters in Strelcyns catalogue.